main.go

```diff
@@ -233,7 +285,7 @@ func prepareIncludedApp(name, source string) (string, error) {
 		return "", err
 	}
 
-	if err := ioutil.WriteFile(filepath.Join(dir, "main.go"), []byte(app.Source), 0644); err != nil {
```

Comment:
How was this working before??

Reply:
I think `app` got pulled in when it's imported on line 23, so it has a property `Source` 🤷